critiquebrainz/frontend/templates/search/selector_results.html

```diff
@@ -66,3 +66,8 @@
 
 {% endif %}
 {% endfor %}
+<script type="text/javascript">
```

Comment:
I think it would be best to move this code to `selector.html` in the `script` block. Also, please add a comment to explain why we are doing this.